templates/zerver/help/add-bots-or-integrations.md

```diff
@@ -0,0 +1,46 @@
+# Add a Bot or Integration
+
+Bots and Integrations are features that are accesible to all members of an Organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.
+
+    ![Settings Dropdown](/static/images/help/cog_dropdown_settings.png)
+
+3. The Settings window will replace your current one. Click on the **Your Bots**
+tab in the column on the left.
+
+    ![Bots Page](/static/images/help/bots_settings_page.png)
+
+4. On this page, you can enter a name, an email, and
+optionally a picture for your bot. Then click **Create Bot** to create your bot.
+
+5. An entry such as the following should appear above the **Add a New Bot** view, confirming
+the creation of your bot.
+
+    ![Example Bot](/static/images/help/bot_example.png)
+
+6. You have succesfully created your bot. You now have access to your bot's API key and API
```

Comment:
Could you possibly explain how to access these things (API key, config file)? There might be another documentation for this, so you can just link it.